index.html

```diff
@@ -0,0 +1,309 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Programming course</title>
+  </head>
+
+  <body>
+    <!--  header -->
+    <header>
+      <ul class="Des-header1">
+        <li class="header1-icon">
+          <a href="#"><img src="./img/facebook.png" alt="facebook logo"/></a>
+        </li>
+        <li class="header1-icon">
+          <a href="#"><img src="./img/twitter.png" alt="twitter logo"/></a>
+        </li>
+        <li class="header1-text">English</li>
+        <li class="header1-text">My Page</li>
+        <li class="header1-text">Logout</li>
+      </ul>
+
+      <div class="main-header">
+        <ul class="humberger">
+          <li class="vector"></li>
+          <li class="vector"></li>
+          <li class="vector"></li>
+        </ul>
+        <ul class="Des-header2">
+          <li>
+            <a href="#" class="page1">
+              <img class="logo" src="./img/logo.png" alt="logo image"
+            /></a>
+          </li>
+          <li><a href="#" class="page1">Home</a></li>
+          <li><a href="#" class="about-page">About</a></li>
+          <li><a href="#program" id="home-page">Program</a></li>
+          <li><a href="#partner" id="home-page">Sponsor</a></li>
+          <li class="logo2">CC Compaign</li>
+        </ul>
```

Comment:
- [ ] [OPTIONAL] Good job so far! It is nice if you add style to your `Navigation menu` elements like [Behance design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) 👍🏼 

![image](https://user-images.githubusercontent.com/63915024/210550316-df889f71-d1dc-4649-9b1e-4823f4d71578.png)